.zenodo.json

```diff
@@ -1,5 +1,5 @@
 {
-  "title": "Alpaka: Abstraction Library for Parallel Kernel Acceleration",
```

Comment:
In case it's published already, should not change. Could you check?

Reply:
This will be the text for the next release and there is no need that this text stay always the same. I also see that we missed to change that it is C++14 and not C++11. (will create a separate PR)